packages/lesswrong/server/resolvers/tagResolvers.ts

```diff
@@ -38,6 +38,9 @@ import { getToCforTag } from '../tableOfContents';
 import { contributorsField } from '../utils/contributorsFieldHelper';
 import { loadByIds } from '@/lib/loaders';
 import { hasWikiLenses } from '@/lib/betas';
+import { randomId } from '@/lib/random';
+import { MultiDocuments } from '@/lib/collections/multiDocuments/collection';
```

Comment:
doesn't need to be imported; can be pulled out of the resolver context in `defineMutation`